src/components/MarvinWindow.vue

```diff
@@ -102,8 +72,10 @@ export default {
     window.addEventListener(
```

Comment:
I think this event listener is persisting on the window and causing an error in the devtools console
`MarvinWindow.vue?1f14:46 Uncaught TypeError: Cannot read property 'postMessage' of null`
that comes up when trying to call `exportMrvfile`. It has caused my browser to crash, when going to the editor and editing, then navigating away from the substance page. Maybe these need to get deleted in the [destroyed](https://vuejs.org/v2/api/#destroyed) life cycle?

Reply:
I've spent some time looking into this.  I believe there are a few different issues.

1. Your crash was caused by an infinite loop.  The "else" used to call `exportMrvfile()` was meant to ensure the editor never failed to have parity with the data in the editor window.  Essentially it would request the mrvfile and the mrvfile response and force another request.  I can't replicate whatever the issue was that this was attempting to fix.  Thanks for catching this!
2. It looks like there is a lot of junk being thrown out by marvin due to the way we are handling this listener on mount.  Rather than the event listener being attached to the component it's being attached to the window.  This seemed to be handled well by removing the method by destroying it.  I'll need to work more with this as it seems marvin is throwing errors now.